test-ts/aave/.waffle.json

```diff
@@ -0,0 +1,11 @@
+{
+    "compilerVersion": "v0.5.13+commit.5b0b510c",
+    "compilerOptions": {
+        "optimizer": {
```

Comment:
As it's for tests only, we can probably do without optimizer.

Reply:
I think we want the tests to reflect the same gas usage as with the optimizer, so we can record gas costs in tests. Ultimately we want to replace the gas profiler with a new one , but for now I think it makes sense to just keep this. tests in typescript don't use the Gas Profile though.